repo_tools_data_schema/repo_tools_data_schema.py

```diff
@@ -99,9 +99,10 @@ def _check(d):
                 Optional('orgs'): [valid_org],
                 Optional('repos'): [valid_repo],
                 Optional('champions'): [existing_person],
+                Optional('branches'): [not_empty_string],
             },
-            # You have to specify eithr orgs or repos:
-            one_of_keys("orgs", "repos"),
+            # You have to specify at least one of orgs, repos, or branches:
+            one_of_keys("orgs", "repos", "branches"),
```

Comment:
@nedbat I messed this up. `one_of_keys` expects exactly one key. I think you should be able to specify `branches` in addition to `repos`. is there a reason why orgs & repos is exclusive? What's the best way to fix this - just check that only one of `orgs` and `repos` exist, and then not worry if `branches` is also specified?

Reply:
Hmm, and specifying repos and branches is an OR, right? You are a committer anywhere in repo A, but also, on any branch named release/*.  I guess we should generalize it, yes.  orgs vs repos didn't have that kind of need.

Reply:
OK. Would you recommend I add an OR check as well, or just leave the existing one-of-keys for orgs & repos?

Reply:
Let's make it orgs OR repos OR branches.  At least one of the three must be specified.

Reply:
https://github.com/edx/repo-tools-data-schema/pull/4